src/client/components/PasswordForm.tsx

```diff
@@ -179,24 +175,22 @@ const ValidationMessage = ({
   isTooShort,
   isTooLong,
   isChecking,
-  noMargin,
 }: {
   isWeak: boolean;
   isTooShort: boolean;
   isTooLong: boolean;
   isChecking: boolean;
-  noMargin?: boolean;
 }) => {
   if (isTooShort) {
-    return <TooShort noMargin={noMargin} />;
+    return <TooShort />;
```

Comment:
was the `noMargin` on these here for a reason? I'm guessing so we could match the designs exactly? from looking at the chromatic diffs there's a small difference between the gap between the input and the button when the lower message is displaying and when it isn't, did my best to capture the difference with a floaty ruler app I found for Mac

![Screenshot 2022-03-09 at 17 23 27](https://user-images.githubusercontent.com/1771189/157496847-dacfcf81-5da4-439c-9a3c-b178321dfac9.png)
![Screenshot 2022-03-09 at 17 24 07](https://user-images.githubusercontent.com/1771189/157496851-ea6760a4-f1db-45b4-a5f0-3eb1b08527ba.png)

Reply:
I don't think it really needs changing if it looks good on mobile/across other viewports, might be worth a quick look at the original Figma designs to make sure though

Reply:
I think it was to account for the fact that the password form needed to reside in two different layouts. The one used in `/welcome` previously needed an extra margin here. Although tbh the styling wasn't very consistent even so, e.g. when there was an error these validation messages didn't show, so the margin they gave was lost and it looked strange (Dom has raised this before).

Reply:
that makes sense, can do some changes for consistency later on down the road then